crates/telio-traversal/src/endpoint_providers/upnp.rs

```diff
@@ -34,18 +34,25 @@ use mockall::automock;
 const MAX_SUPPORTED_PACKET_SIZE: usize = 1500;
 const GET_INTERFACE_TIMEOUT_S: Duration = Duration::from_secs(2);
 
+const MIN_EPHEMERAL_PORT_RANGE: u16 = 49152;
+const MAX_EPHEMERAL_PORT_RANGE: u16 = 65535;
+
+const DEFAULT_LEASE_DURATION_S: u32 = 600;
```

Comment:
It's nice that it's postfixed with `_S` but I think it could benefit from being `time::Duration`, this way it would be more type-safe

Reply:
fair enough, will change